project/CopyrightHeader.scala

```diff
@@ -12,8 +12,8 @@
  */
 
 import sbtheader.{ CommentCreator, HeaderPlugin, NewLine }
-import org.apache.commons.lang3.StringUtils
-import sbt._
+import org.apache.commons.lang3.Strings
+import sbt.*
```

Comment:
can we keep the _ syntax?

Reply:
Yes, but we should migrate to this sooner or later.